src/util.ts

```diff
@@ -81,6 +81,11 @@ let vendorSupport: boolean = null;
 let telemtryReporter: TelemetryReporter;
 let toolsGopath: string;
 
+// Assumes the Go version is always 1.x.
+export function isBelow(goVersion: SemVersion, minor: number): boolean {
```

Comment:
I'd prefer that this function take `major` as an argument and skip the assumption that the major version is always 1

Reply:
Done.